File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.py
class Solution:
    def differenceOfSum(self, nums: List[int]) -> int:
        ele = sum(nums)
        dig = 0
        
        for i in nums:
            while i > 0:
                dig += i % 10
                i //= 10
                
        return abs(ele - dig)

File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.java
class Solution {
    public int differenceOfSum(int[] nums) {
        int ele = 0;
        int dig = 0;
        
        for (int i : nums){
            ele += i;
            
            int temp = i;
            while (temp > 0){
                dig += (temp % 10);
                temp /= 10;
            }
        }
        return Math.abs(ele - dig);
    }
}